src/main/java/pageObjects/DashboardPage.java

```diff
@@ -0,0 +1,32 @@
+package pageObjects;
+
+import driver.DriverFactory;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class DashboardPage extends DriverFactory {
+    public static String URL = "https://zenistock.zenitech.local/dashboard";
+    public static String TagsURL = "https://zenistock.zenitech.local/tags";
+
+    @FindBy(id = "undefined-tags-mButton")
+    private WebElement tagsMenu;
+
+    public DashboardPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void goToDashboard() {
+        PageFactory.initElements(driver, this);
+        driver.get(URL);
+        waitForURL(URL);
+    }
+
+    public void goToTags() {
+        goToDashboard();
+        waitForElementToBeClickable(tagsMenu);
+        tagsMenu.click();
+        waitForURL(TagsURL);
+    }
+}
```

Comment:
since we currently have nothing on the dashboard page, i dont think we need this class at the moment. for your tests you can just login and click on the tags page sidemenu button

Reply:
i guess we will we have at one point and the things here don`t really belong in the other classes

Reply:
I can agree with @CaprarClaudiu , we should not commit this file at the moment, later if we need it, we will do a separate PR for it.